src/axom/spin/internal/linear_bvh/BVH_impl.hpp

```diff
@@ -55,6 +59,9 @@ using vec4_t = internal::linear_bvh::Vec< FloatType, 4 >;
 template < typename FloatType, int NDIMS >
 using point_t = internal::linear_bvh::Vec< FloatType, NDIMS >;
 
+template < typename FloatType, int NDIMS >
+using ray_t = internal::linear_bvh::Vec< FloatType, NDIMS*2 >;
```

Comment:
Is this the ``RayType`` for ``getRay()`` below?

Reply:
Yes, that's what I use for `RayType`.